src/styles/components/_product-information.scss

```diff
@@ -1,20 +1,122 @@
 .product-information {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
   box-sizing: border-box;
-  padding: rem(16px);
+  padding: rem(18px 24px);
+
+  @include respond-above('md') {
+    padding: 0;
+  }
+
+  &__header,
+  &__body {
+    @include respond-above('md') {
+      background-color: $alabaster;
+    }
+  }
 
   &__header {
-    margin-bottom: rem(24px);
+    display: flex;
+    align-items: center;
+
+    @include respond-above('md') {
+      width: rem(200px);
+      padding: rem(20px) 0 0 rem(28px);
+      border-top-left-radius: rem(32px);
+    }
+
+    @include respond-above('lg') {
+      width: rem(220px);
+      padding: rem(24px) 0 0 rem(32px);
+    }
+  }
+
+  &__subheader {
+    display: none;
+
+    @include respond-above('md') {
+      display: block;
+      margin-bottom: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      margin-bottom: rem(24px);
+    }
   }
 
   &__title {
-    margin: 0 0 rem(16px) 0;
+    all: unset;
```

Comment:
Quick question, if this is just to reset the `margin`, does it necessary to unset **all**?
PS. Even `description` doesn't have this.

Reply:
Not just `margin` but also `font-weight` since `h2` is bold by default.

With the override `margin` and `font-weight`
<img width="341" alt="Screen Shot 2564-03-28 at 00 02 30" src="https://user-images.githubusercontent.com/32285706/112728268-fe396380-8f58-11eb-89e9-740f022f1ca4.png">

Do you think this one is cleaner?
<img width="359" alt="Screen Shot 2564-03-28 at 00 01 46" src="https://user-images.githubusercontent.com/32285706/112728271-009bbd80-8f59-11eb-873e-20301febd58a.png">

Reply:
Agree with using unset.